Modules/CalcOffence-3_0.lua

```diff
@@ -1016,7 +1016,8 @@ function calcs.offence(env, actor, activeSkill)
 
 		-- Calculate Double Damage + Ruthless Blow chance/multipliers
 		output.DoubleDamageChance = m_min(skillModList:Sum("BASE", cfg, "DoubleDamageChance") + (env.mode_effective and enemyDB:Sum("BASE", cfg, "SelfDoubleDamageChance") or 0), 100)
-		output.DoubleDamageEffect = 1 + output.DoubleDamageChance / 100
+		output.CritDoubleDamageChance = m_min(skillModList:Sum("BASE", cfg, "CritDoubleDamageChance"), 100)
+		output.DoubleDamageEffect = 1 + (output.DoubleDamageChance + (output.CritDoubleDamageChance*output.CritChance/100)) / 100
```

Comment:
IIRC there are separate code paths for crit and non crits in CalcOffesnse, which are important due to a bunch of other conditionals which depend on crits.

Reply:
are you saying this is wrong or that you need more verification that this change is correct?

Reply:
I believe this is wrong - although I still might be missing something.
In the scenario where you deal `non_crit_damage` and `crit_damage` the total expected dps should be `crit_chance * (crit_double_damage_multi) * crit_damage + (1- crit_chance) * (non_crit_double_damage_multi) * non_crit_damage`

I believe the way you have it written is `[crit_chance * (crit_double_damage_multi) + (1- crit_chance) * (non_crit_double_damage_multi)]* [crit_chance  * crit_damage + (1- crit_chance) * non_crit_damage]`